kotlin-reference-server/src/main/kotlin/org/stellar/reference/event/processor/Sep6EventProcessor.kt

```diff
@@ -0,0 +1,204 @@
+          )
+        activeTransactionStore.removeTransaction(customer.id, transaction.id)
+        log.info(
+          "Removed transaction ${transaction.id} from active transaction store for customer ${customer.id}"
+        )
+      }
+      else -> {
+        log.warn("Received transaction status changed event with unsupported status: $status")
+      }
+    }
+  }
+
+  override fun onCustomerUpdated(event: AnchorEvent) {
+    val observedAccount = event.customer.id
+    val transactionIds = activeTransactionStore.getTransactions(observedAccount)
+    log.info(
+      "Found ${transactionIds.size} transactions for customer $observedAccount in active transaction store"
+    )
+    transactionIds.forEach { id ->
+      val transaction = runBlocking { platformClient.getTransaction(id) }
+      if (transaction.status == SepTransactionStatus.PENDING_CUSTOMER_INFO_UPDATE) {
+        val keypair = KeyPair.fromSecretSeed(config.appSettings.secret)
+        val assetCode = toStandardFormat(transaction.amountOut.asset)
+
+        val asset = Asset.create(assetCode)
+        val amount = transaction.amountOut.amount
+        val destination = transaction.destinationAccount
+
+        val stellarTxn = submitStellarTransaction(keypair.accountId, destination, asset, amount)
+        runBlocking {
+          patchTransaction(
+            PlatformTransactionData.builder()
+              .id(transaction.id)
+              .status(SepTransactionStatus.COMPLETED)
+              .updatedAt(Instant.now())
+              .completedAt(Instant.now())
+              .requiredInfoMessage(null)
+              .requiredInfoUpdates(null)
+              .requiredCustomerInfoUpdates(null)
+              .requiredCustomerInfoUpdates(null)
+              .instructions(
+                mapOf(
+                  "organization.bank_number" to
+                    InstructionField.builder()
+                      .value("121122676")
+                      .description("US Bank routing number")
+                      .build(),
+                  "organization.bank_account_number" to
+                    InstructionField.builder()
+                      .value("13719713158835300")
+                      .description("US Bank account number")
+                      .build(),
+                )
+              )
+              .stellarTransactions(listOf(stellarTxn))
+              .build()
+          )
+        }
+      }
+    }
+  }
+
+  private fun toStandardFormat(asset: String): String {
+    val parts = asset.split(":")
+    return when (parts.size) {
+      3 -> "${parts[1]}:${parts[2]}"
+      2 -> parts[1]
+      else -> throw RuntimeException("Invalid asset format: $asset")
+    }
+  }
+
+  private fun submitStellarTransaction(
+    source: String,
+    destination: String,
+    asset: Asset,
+    amount: String
+  ): StellarTransaction {
+    val account = server.accounts().account(source)
```

Comment:
[nit] you can use wallet SDK for it, it will be 5 lines of code instead of 20

Reply:
Ah, nice. I'll mark this as a TODO for now and get to it in ANCHOR-439.